rakuten/query-params-audience/README.md

```diff
@@ -0,0 +1,37 @@
+# Query Params Audience
+Create an audience based on people who have visited your site with specific query parameters. You can use this to identify paid traffic.
+
+This integration sets the local storage key `evolvParamStore` to a set value provided in the Environment's integration configuration JSON.
+
+1. Add the integration to your account.
+2. Open the environment where you want to use the integration.
+3. Select the integration from the list of available integrations.
+4. Set the configuration JSON.
+
+## Configuration JSON
+```
+{
```

Comment:
This design looks like it allows for one audience value per integration.

If they wanted something like:
Set the source of the traffic based on query params
utm_campaign=google Sets context to traffic_source = google
utm_campaign=facebook Sets traffic_source = facebook

I don't think this could handle it. I'd be tempted to create a general integration that you can configure to handle multiple context values be set dependent on query params

Not a blocker for this PR, as I know Rakuten may need this - but there is a general integration that would be useful

Also worth adding a test